src/bindings/js/node/tests/unit/compiled_model.test.js

```diff
@@ -0,0 +1,78 @@
+// -*- coding: utf-8 -*-
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+const { addon: ov } = require('../..');
+const assert = require('assert');
+const { describe, it } = require('node:test');
+const { getModelPath } = require('./utils.js');
+
+const testXml = getModelPath().xml;
+const core = new ov.Core();
+const properties = {
+  "AUTO_BATCH_TIMEOUT": '1'
+};
+const compiledModel = core.compileModelSync(testXml, 'BATCH:CPU', properties);
+
+describe('setProperty() / getProperty()', () => {
+
+  describe('getProperty()', () => {
+    it('returns the value of property from compiled model', () => {
+      assert.strictEqual(compiledModel.getProperty('AUTO_BATCH_TIMEOUT'), '1');
+    });
+    it('throws an error when called without arguments', () => {
+      assert.throws(
+        () => compiledModel.getProperty(),
+        /Expected a single string argument to retrieve property value/
```

Comment:
Please remember to update the error message after changes to arguments validation ;)